sdk-actors/src/main/java/io/dapr/actors/runtime/DaprStateAsyncProvider.java

```diff
@@ -68,6 +68,19 @@
           return Mono.empty();
         }
 
+        if (!this.isStateSerializerDefault) {
+          if (s.length == 0) {

```

Comment:
Why is this needed only in the deserialize path?

Reply:
In another way we'll get an exception